scripts/search/settings.js

```diff
@@ -104,8 +107,36 @@ module.exports = {
                     promote: [
                         {
                             objectIDs: [
-                                // The Pulumi Cloud overview page.
-                                "a2074d4bcd83fa3d347dc9da4b8aa822"
+                                page.getObjectID({ href: "/docs/pulumi-cloud/" }),
+                            ],
+                            position: 0,
+                        },
+                    ],
+                    filterPromotes: true,
+                },
+            },
+
+            // When the query is for "output" or "outputs", deliver the Inputs & Outputs page as the top result.
```

Comment:
does this need to be done for querying the word "input" too, or is it only output that turns up other results?

Reply:
Actually it turns out we don't need this when we change the behavior of the `title` attribute to be `unordered`, which I've done here instead. (Didn't feel great having to do this, so hopefully that'll work in general. It's [documented here](https://www.algolia.com/doc/guides/managing-results/must-do/searchable-attributes/how-to/configuring-searchable-attributes-the-right-way/#understanding-word-position), btw.)